### ISSUE TITLE
netserver: Crash on website load in WebKitGTK

### ISSUE THREAD

**Issue by Dennisbonke** [managarm/managarm#431]
When starting the MiniBrowser, it will try to load the webkitgtk home page (tho I could reproduce it with other webpages too). That will eventually crash the minibrowser.

Backtrace:
```
(gdb) bt
#0  std::__uniq_ptr_impl<virtio_core::(anonymous namespace)::LegacyPciQueue, std::default_delete<virtio_core::(anonymous namespace)::LegacyPciQueue> >::_M_ptr (this=0x9)
    at /mnt/linux/managarm/managarm-clone/build/tools/system-gcc/lib/gcc/x86_64-managarm/11.1.0/../../../../x86_64-managarm/include/c++/11.1.0/bits/unique_ptr.h:173
#1  std::unique_ptr<virtio_core::(anonymous namespace)::LegacyPciQueue, std::default_delete<virtio_core::(anonymous namespace)::LegacyPciQueue> >::get (this=0x9)
    at /mnt/linux/managarm/managarm-clone/build/tools/system-gcc/lib/gcc/x86_64-managarm/11.1.0/../../../../x86_64-managarm/include/c++/11.1.0/bits/unique_ptr.h:422
#2  std::unique_ptr<virtio_core::(anonymous namespace)::LegacyPciQueue, std::default_delete<virtio_core::(anonymous namespace)::LegacyPciQueue> >::operator-> (this=0x9)
    at /mnt/linux/managarm/managarm-clone/build/tools/system-gcc/lib/gcc/x86_64-managarm/11.1.0/../../../../x86_64-managarm/include/c++/11.1.0/bits/unique_ptr.h:416
#3  virtio_core::(anonymous namespace)::LegacyPciTransport::_processIrqs (this=<optimized out>) at ../../../src/managarm/core/virtio/src/core.cpp:247
#4  0x0000000000450d86 in helix::Dispatcher::wait (this=0x7fffffc3c240) at ../../../src/managarm/hel/include/helix/ipc.hpp:262
#5  0x00000000004434ad in helix::CurrentDispatcherToken::wait (this=<optimized out>) at ../../../src/managarm/hel/include/helix/ipc.hpp:350
#6  async::run_forever<helix::CurrentDispatcherToken> (ios=...) at /mnt/linux/managarm/managarm-clone/build/system-root/usr/include/async/basic.hpp:305
#7  main () at ../../../src/managarm/servers/netserver/src/main.cpp:193
(gdb)
```

Relevant crash message in E9:
```
received request for socket creation
netserver:
 f020a0a
 9090909
    2b00
    92eb
    3500
    2b00
    4f3e
virtio-driver: sending frame
thor: Unhandled page fault at 0x9, faulting ip: 0x43829fb5
thor: Page fault at 0x9, faulting ip: 0x43829fb5
Errors: (User) (Page not present) (Read)
thor: Page fault in server, at 0x9, faulting ip: 0x43829fb5
thor: Fault in server /usr/bin/netserver
thor: Launching gdbserver on I/O channel kernel-gdbserver
```

**Comment by qookei** [comment]
Wasn't this fixed by e6c9c6bf4e0a4c40168fd44a7a32ed046079cb2c?

**Comment by Dennisbonke** [comment]
I believe it was qookie, closing this as I can no longer reproduce it.
